workflow-Estimate/workflow/dependencies/dahaner2017_liu2015_immune_genesets.gmt

```diff
@@ -0,0 +1,66 @@
+Hematopoietic_stem_cell_CD133+_CD34dim	GSE24759_21_Liu2015	TRIP6	FSTL1	PFN2	SPINK2	FARP1	MLC1	PTPLA	HDGFRP3	ENO1	CD34	TFEC	ATP7B	MPDZ	SCN3A	STK32B	PPM1H	EPLIN	PLSCR4	WASF1	PLCXD1	HSPB1	DDEF2	SOCS2	IMP-2	MPL	CEP70	MGC3032	HYMAI	DOK4	GGH	MPPED2	CXCL3	NPTX2	CHRDL1	BTBD3	TBC1D4	DTL	EMID1	KIAA0125	VRK2	GUCY1A3	GPR126	MGC39900	FNBP1L	COL4A5	SH3BP4	CLGN	ERG	FGF13	HMGB3	ATP8B4	MEIS1	NGFRAP1	PLS1	PDGFD	MEST	PAWR	PAICS	SOX4	MN1	PTPRD	ME3	SNRPD1	FADS1	CRYGD	TSC22D1	ITGAV	NPM3	BST2	HOXA10	ZNF261	LMO2	TNFSF4	KIT	SLC39A14	HTR1F	GATA2	NAP1L3	SLC35F2	SCD	MAP7	SERPINE2	ZNF323	PKD2	PRSS2	MYCT1	WBP5	PLCB1	PLCB4	SEPP1	NR1H3	NUDT11	ROBO1	SPAG16	SPP1	MYH10	DPY19L4	HIST1H2BG	DACH1	KIAA0649	FZD6	DLG5	MANSC1	SPRY1	UCHL1	LOC400451	TMEM45A	EHD2	CYTL1	NPR3	SEPTIN11	HOXA5	NBEA	GNAI1	LOC63920	KLHL23	IL1B	ZNF167	LRRC16	KIAA1815	CHRM3	IL18	SALL2	FSCN1	HIST2H2AA	DKFZP586D0919	FLT3	GSTM5	FLJ14054	DKFZP761N09121	HIST1H2AE	C5ORF13	FLJ13197	POLE2	NET1	TOX	BCL11A	CLDN10	RPL31	TRPC1	DEPDC6	FLJ22746	ITM2C	SCCPDH	GSTA4	MMRN1	EREG	LST1	AKR1C3	F2RL1	PLAG1	COL5A1	DDAH1	FKBP9	FLJ13646	SERPING1	TENS1	ALDH7A1	TSPYL5	KIAA1102	PXDN	BAALC	KCTD3	FANCL	BSPRY	DPPA4	MTL5	CRHBP	PTRF	MGST2	DNMT3B	AREG	CDC42BPA	PLEKHA5	ANKRD28	LDB2	CPA3	IGF1R	C21ORF25	TCEAL4	NR4A3	ALDH1A1	SERPINI2	ARMCX1	BHLHB9	ASB9	MLLT3	PHF16	FLJ11259	PLS3	TNFRSF21	THNSL1	CFH	NRIP1	ETS2	PGDS	PLOD2	DSU	GATM	HOP	C14ORF114	NME1	RHOBTB1	LOC93081	KLHL3	KDELC1	EGR3	DSG2	DAPK1	VAV3	EXTL2	ARHGAP22	LOH11CR2A	KIAA1305	C1ORF54	DBN1	BEX1	LAPTM4B	KIAA1447	HLF	TMEM22	PDE1A	LHFP	SPRY2	PROM1	FLJ36166	HIST2H2BE	JARID1B	SSBP2	C14ORF110	TARBP1	PDGFC	ANGPT1	MLC1SA	GPR56	HOXA9	H1F0	IGLL1	SMARCA1	HMGA2	SCARF1	HBB	GNG11	FHL1	RAB13	PBX1	RNF144	HBG1	HBA2	PLAGL1	LOC90355	KIF13A	MCTP1	CBFA2T3	PDZK3	SCHIP1						

```

Comment:
Did you really want to check this file in? Also the ensembl file in the same directory.